zipline/finance/performance/tracker.py

```diff
@@ -191,6 +188,10 @@ def __init__(self, sim_params, env=None):
         self.account_needs_update = True
         self._account = None
 
+        self._known_asset_sids = set()
```

Comment:
I cache the sids whose assets have already been pulled in order to avoid pulling the same Asset out of the finder. If the sid is in the set _known_asset_sids, I know that the necessary fields have already been populated.